recipes/blas/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set blas_pkg = "openblas" %}
+
+{% set version = "1.0.0" %}
+{% set build_number = 0 %}
+
+{% set version_post = (".post" + build_number|string) if build_number|int != 0 else "" %}
+{% set version_full = version + version_post %}
```

Comment:
> I really would encourage you not to strive so hard to treat these like normal packages. They're definitely not, and it would be just fine to define a separate standard for them.

Yeah, so, I'm kind of drawing attention to this so people here also agree this is an ugly mess and figure out some better way here or in our infrastructure to address this.

> In fact, if this works well, we may bury some of these details into conda syntactic sugar.

That would be awesome. Maybe it can be what features simply weren't.